### PR TITLE
Databricks Volumes v2 destination connector

### PR DESCRIPTION
**Pull request by Paul-Cornell**
See:

- API: https://unstructured-53-docs-36-databricks.mintlify.app/api-reference/ingest/destination-connector/databricks-volumes
- Open source: https://unstructured-53-docs-36-databricks.mintlify.app/open-source/ingest/destination-connectors/databricks-volumes

### CODE REVIEW

File: snippets/general-shared-text/databricks-volumes.mdx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+The Databricks Volumes prerequisites:
```
Comment (MKhalusova) [inline snippets/general-shared-text/databricks-volumes.mdx:1]: This snippet doesn't show in staging. Can you make sure that it's imported correctly?

**Review by MKhalusova — CHANGES_REQUESTED** [review]

File: snippets/general-shared-text/databricks-volumes.mdx (hunk 1 above)
Comment (Paul-Cornell) [reply snippets/general-shared-text/databricks-volumes.mdx:1]: It looks fine with my local `mintlify dev` but not here for some reason?

Comment (Paul-Cornell) [reply snippets/general-shared-text/databricks-volumes.mdx:1]: Would you be willing to pull this branch locally and run `mintlify dev` as well to double-check?

Comment (Paul-Cornell) [reply snippets/general-shared-text/databricks-volumes.mdx:1]: If this is still an issue, I can ask the Mintlify team to weigh in on this maybe?

Comment (MKhalusova) [reply snippets/general-shared-text/databricks-volumes.mdx:1]: It builds properly for me locally as well

**Review by MKhalusova — APPROVED** [review]

**Review by ninalopatina — APPROVED** [review]

**Pull request merged**